tools/percolator/metafiles2pin.xml

```diff
@@ -1,116 +0,0 @@
-<tool id="metafiles2pin" name="Merge search data" version="1.0">
-    <description>before creating percolator input</description>
-    <stdio>
```

Comment:
Yes, I have removed the metafiles and they are now supplied as nested collections :)